code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -431,27 +431,33 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 			var/datum/gas_reaction/reaction = r
 
 			var/list/min_reqs = reaction.min_requirements.Copy()
-			if((min_reqs["TEMP"] && temp < min_reqs["TEMP"]) \
-			|| (min_reqs["ENER"] && ener < min_reqs["ENER"]))
+			var/min_temp = min_reqs["TEMP"]
+			var/min_ener = min_reqs["ENER"]
+			if((!isnull(min_temp) && temp < min_temp) \

```

Comment:
The line is not even long. I just have the opinion that multilines are cancer and should be avoided as much as possible.